modules/tensor_mechanics/include/materials/StateUpdateBase.h

```diff
@@ -0,0 +1,82 @@
+/****************************************************************/
```

Comment:
Again, since all of the models that derive from this are called `*StressUpdate`, I think we should keep the existing name.